src/app/data/loaders/biologicLoader.ts

```diff
@@ -3,23 +3,32 @@ import { convert } from 'biologic-converter';
 import type { FileCollection } from 'filelist-utils';
 
 import { getEmptyMeasurements, Loader } from '../DataState';
+import type { MeasurementBase } from '../MeasurementBase';
 
 export const biologicLoader: Loader = async function biologicLoader(
   fileCollection: FileCollection,
 ) {
   let measurements = getEmptyMeasurements();
   const results = await convert(fileCollection);
-  for (let result of results) {
+  for (let { dir, mpr, mps, mpt } of results) {
     //still not for plotting, just a schema
-    measurements.iv.entries.push({
+    const prepare: Partial<MeasurementBase> = {
       id: v4(),
-      meta: result.mps ? { ...result.mps } : {},
       filename: '',
-      path: result.dir,
+      path: dir,
       info: {},
       title: '',
-      data: [],
-    });
+    };
+    if (mpr !== undefined) {
+      prepare.meta = { settings: mpr.settings, log: mpr?.log };
```

Comment:
For meta it is better if it is flat and the log is not that important.
Something like:
`prepare.meta = {...mpr.settings)`
We will display those values and it will be searchable in the following pannel: https://zakodium-oss.github.io/analysis-ui-components/stories/?story=measurements--panels--measurement-info-panel

Reply:
It should be done now. Are you running `npm run dev-app` as well? I wonder if why would we see different output, could be some update in between.